corehq/motech/generic_inbound/reports.py

```diff
@@ -100,7 +102,7 @@ def rows(self):
         for log in self._queryset[self.pagination.start:self.pagination.end]:
             yield [
                 log.api.name,
-                log.timestamp,
+                ServerTime(log.timestamp).user_time(self.timezone).ui_string(),
```

Comment:
nice, I didn't notice the `UserTime.ui_string` method when localizing the timestamp on the API keys page.  Good to know about.